modules/cpmso-yaml-failure-domain-aws.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-aws_{context}"]
+= Sample AWS failure domain configuration
+
+The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+
+.Sample AWS failure domain values
+[source,yaml]
+----
+failureDomains:
+  aws:
+  - placement:
+      availabilityZone: <aws_region_1> <a>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_1> <c>
+      type: Filters
+  - placement:
+      availabilityZone: <aws_region_2> <d>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_2> <e>
+      type: Filters
+  platform: AWS <f>
+----
+<a> Specifies an AWS region for the first failure domain.
+
+<b> #Does this need any information?#
+
+<c> Specifies the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster and the AWS region for the first failure domain.
+
+<d> Specifies an AWS region for an additional failure domain.
+
+<e> Specifies the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster and the AWS region for the additional failure domain.
+
+<f> Specifies the cloud provider platform name. Do not change this value.
+
+//Does the user really "specify" any of this? Not sure what is at their discretion vs. what needs to be copied from an existing set and left alone.
```

Comment:
If they are configuring it manually (which on AWS is unlikely to be the case) then yes, otherwise it should be auto created for them but it's useful to understand the structure I would say

In the future they may want to change the failure domains, move their cluster from a,b,c to d,e,f in which case this is helpful